history.txt

```diff
@@ -1,6 +1,38 @@
 OMERO version history
 =====================
 
+5.2.0 (November 2015)
+----------------------
+
+A full, production-ready release of OMERO 5.2.0; dropping support for Java
+1.6; featuring major upgrading of OMERO.web; re-working of the Java Gateway;
+and introducing new user features and many fixes and performance improvements:
+
+-  improved support for many file formats via Bio-Formats
```

Comment:
Sorry for the late comment, should we start including specific Bio-Formats versions in the release?